.mailmap

```diff
@@ -350,7 +350,8 @@ Kévin Caradant <kevin.caradant@gmail.com>
 Lan De <lan.de3@zte.com.cn>
 Lan Liu <liulan@umcloud.com> <rjerk.whatever@gmail.com>
 Laszlo Boszormenyi <gcs@debian.hu>
-Laura Flores <lflores@redhat.com>
+Laura Flores <lflores@redhat.com> <ljflores@redhat.com>
+Laura Flores <lflores@redhat.com> Laura  Flores <lflores@redhat.com>
```

Comment:
Not sure if it matters, but there's an extra space between "Laura" and "Flores" on line 354.